schemas/create-status.yml

```diff
@@ -0,0 +1,22 @@
+$schema:  http://json-schema.org/draft-04/schema#
+title:        "Create status"
+description: |
+  Create a commit status on GitHub
+type:         object
+properties:
+  state:
+    type: string
+    enum: ['pending', 'success', 'error', 'failure']
+    description: github status associated with the build.
```

Comment:
Let's make the descriptions full sentences starting with capital letter and ending with period. Not sure we do that correctly everywhere already, but may as well do it here!